components/artwork_item/templates/artwork.jade

```diff
@@ -19,6 +19,8 @@ figure.artwork-item( data-artwork= artwork.get('id') data-id= artwork.get('_id')
       include ./partials/save
 
   figcaption.artwork-item-caption
+    if hasAuctionPartner && !isAuction
```

Comment:
👍 to checking `sale_ids`. It looks like `isAuction` is kinda a hack that is probably irrelevant with this redesign. I believe we used `sets` on the old auction pages (even before the current design).

🔥  ordered sets 🔥  🔫